provider.tf

```
@@ -0,0 +1,10 @@
+terraform {
+  required_providers {
+    # https://registry.terraform.io/providers/orange-cloudavenue/cloudavenue/latest/docs
+    cloudavenue = {
+      source  = "orange-cloudavenue/cloudavenue"
+      version = ">=0.31.0"
```

```suggestion
      version = ">=0.32.0"
```
Need the new version of TF provider cav